starknet_py/common.py

```diff
@@ -31,6 +40,23 @@ def int_from_hex(number: Union[str, int]) -> int:
     return number if isinstance(number, int) else int(number, 16)
 
 
+def get_selector_from_name(func_name: str) -> int:
```

Comment:
Can we move all those functions to a better place?

Reply:
Not sure what would be the best place for this.
`starknet_py.net.client_utils` maybe?
Or a new separate file?

Reply:
I think cairo/selector makes sense. Selector is a part of the cairo program.

Reply:
What about `get_storage_var_address`?

Reply:
`cairo/storage.py`? I like grouping things properly. @war-in how do you plan to arrange something like this in changed structure?